src/app/Component/ChatScreen/chatscreen.css

```diff
@@ -57,13 +57,13 @@
 {
     border-radius: 90%;
     margin-top: 6px;
-    margin-left: 450px;
+    margin-left: 550px;
```

Comment:
Can you use left or right css properties instead of margin-left her to align element as it's not a good coding practise to give margin values beyond 25px usually.